src/components/ButtonGroup/index.jsx

```diff
@@ -5,33 +5,17 @@ import GroupButton from './GroupButton';
 
 const ButtonGroup = ({ buttons, onChange, ...restBtnGroup }) => (
   <div {...restBtnGroup} className="btngroup">
-    {buttons.map(
-      (
-        {
-          value,
-          content,
-          key,
-          Icon,
-          onClick,
-          label,
-          // `isFirstButton` and `isLastButton` are private and can't being passed in.
-          isFirstButton,
-          isLastButton,
```

Comment:
These two props aren't getting carried over?

Reply:
When I updated eslint, it complained that these values were not being used, which I don't think they are.